src/toil/common.py

```diff
@@ -554,6 +556,10 @@ def _addOptions(addGroupFn, config):
                 help=("If set, preemptable failures (or any failure due to an instance getting "
                       "unexpectedly terminated) would not count towards job failures and "
                       "--retryCount."))
+    addOptionFn("--doubleMem", dest="doubleMem", action='store_true', default=False,
```

Comment:
Since this adds another option it should also update https://github.com/DataBiosphere/toil/blob/c526b6fc20a81b9e04bad46d14840511666d8c3e/docs/running/cliOptions.rst